website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -7,6 +7,12 @@ page_id: api-changelog
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+### 2022-09-21
+* Updated the [Policies API](/cloud-docs/api-docs/policies.mdx) with support for OPA policies`
```

Comment:
Let's define this since it's an acronym - just the first time it's used!

```suggestion
* Updated the [Policies API](/cloud-docs/api-docs/policies.mdx) with support for Open Policy Agent (OPA) policies`
```